examples/tests/sdk_tests/resource/call.test.w

```diff
@@ -0,0 +1,221 @@
+bring expect;
+bring sim;
+
+enum MyEnum {
+  A,
+  B,
+  C
+}
+
+struct MyStruct {
+  field1: str;
+  field2: num;
+}
+
+inflight class MyResourceBackend impl sim.IResource {
+  var field1: str;
+  var field2: (str): str;
+  var ctx: sim.IResourceContext?;
+  new() {
+    this.field1 = "hello";
+    this.field2 = (name: str) => { return "Hello, " + name; };
+    this.ctx = nil;
+  }
+
+  pub onStart(ctx: sim.IResourceContext) {
+    this.ctx = ctx;
+  }
+  pub onStop() {}
+
+  pub throwsError() {
+    throw "Look ma, an error!";
+  }
+
+  pub printLogs() {
+    this.ctx!.log("an info log");
```

Comment:
What happens if I just write `log()` without going through the context? Ideally we should capture these logs and it should Just Work ™️

Reply:
Regular logs work as well, and the messages are treated as INFO level.